Makefile

```diff
@@ -1,4 +1,10 @@
-.PHONY: piptools upgrade fix_transifex_resource_names transifex_resources_requirements validate_translation_files
+.PHONY: piptools upgrade fix_transifex_resource_names transifex_resources_requirements validate_translation_files \
+sync_translations sync_translations_github_workflow
+
+
+# Default languages for the sync_translations.py file
+export TX_LANGUAGES := ar,de,fr_CA
```

Comment:
Eventually, this should be all the languages Open edX supports.

Reply:
it would probably be good to add a comment directly in the file documenting that